extensions/test-extension/src/custom-context-menu/contextMenuCodeItem.ts

```diff
@@ -0,0 +1,27 @@
+import codingValues from './codingValues';
+
+const codeMenuItem = {
+  id: '@ohif/contextMenuAnnotationCode',
+  codingValues,
+
+  /** Applies the code value setup for this item */
+  applyType: function (customizationService) {
```

Comment:
can we come up with a better name for `applyType`? maybe along `transform` (since it is basically transforming)